internal/models/payment_status.go

```diff
@@ -17,6 +17,8 @@ const (
 	PAYMENT_STATUS_FAILED
 	PAYMENT_STATUS_EXPIRED
 	PAYMENT_STATUS_REFUNDED
+	PAYMENT_STATUS_REVERSED
+	PAYMENT_STATUS_RETURNED
```

Comment:
Sorry I hadn't noticed that you had added new payment statuses. Since this needs to be synchronized with the frontend and SDKs, let's not add new statuses.

Both of these can be registered as `PAYMENT_STATUS_REFUNDED`.